Signal/src/AppDelegate.m

```diff
@@ -135,7 +135,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     // Prevent the device from sleeping during database view async registration
     // (e.g. long database upgrades).
     //
-    // This block will be cleared in databaseViewRegistrationComplete.
+    // This block will be cleared in storageIsReady.
     [DeviceSleepManager.sharedInstance addBlockWithBlockObject:self];
```

Comment:
`OWSStorage.isStorageReady` is YES when _ALL_ storage classes have completed both their sync _AND_ async view registrations.